src/staticanalysis/bot/static_analysis_bot/cli.py

```diff
@@ -74,12 +74,9 @@ def main(source,
                 )
 
     # Setup settings before stats
-    settings.setup(
-        secrets['APP_CHANNEL'],
-        cache_root,
-        secrets['PUBLICATION'],
-        secrets['ALLOWED_PATHS'],
-    )
+    settings.setup(secrets['APP_CHANNEL'], cache_root, secrets['PUBLICATION'],
+                   secrets['ALLOWED_PATHS'], secrets['COVERITY_CONFIG']
+                   if 'COVERITY_CONFIG' in secrets else None)
```

Comment:
Instead of `secrets['COVERITY_CONFIG'] if COVERITY_CONFIG' in secrets else None`, you can do `secrets.get('COVERITY_CONFIG')`: that's exactly the same result